backend/src/schema.js

```diff
@@ -0,0 +1,22 @@
+import { stitchSchemas } from '@graphql-tools/stitch';
+import { applyMiddleware } from 'graphql-middleware';
+import { typeDefs } from './typeDefs';
+import { permissions } from './datasource/permissions';
+import Resolvers from './resolvers';
+import GraphCmsSchema, { executor } from './graphCms/schema';
+
+export default async () => {
+  const graphCmsSchema = await GraphCmsSchema();
+  const resolvers = Resolvers({ schema: graphCmsSchema, executor });
```

Comment:
Here you can remove certain fields from the resulting schema: https://github.com/Systems-Development-and-Frameworks/homework/blob/demo/demo/backend/src/schema.js#L11

Reply:
![2021-01-15_16-08](https://user-images.githubusercontent.com/2110676/104743404-f33da800-574b-11eb-9b28-98ab2207d74f.png)